docs/src/apps/sysvars.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Sysvar Cluster Data
+---
+
+Solana exposes a variety of cluster state data to programs via
+[`sysvar`](terminology.md#sysvar) accounts. These accounts are populated at
+known addresses published along with the account layouts in the
+[`solana-sdk`](https://docs.rs/solana-sdk/VERSION_FOR_DOCS_RS/solana_sdk/sysvar/index.html),
```

Comment:
Post-merge review!  Please use `solana-program` instead of `solana-sdk`, which is where these now live as of 1.4

Reply:
Okay, will update